addons/website/static/src/scss/website.scss

```diff
@@ -1400,3 +1400,11 @@ $ribbon-padding: 100px;
         bottom: 0;
     }
 }
+
+.o_website_btn_loading {
+    cursor: wait;
+    opacity: $btn-disabled-opacity;
```

Comment:
Works too I guess :) The only drawback is that the Odoo UI buttons now depends on the theme... but that's ok that would not be the first stuff :wink:

Reply:
Let's keep it that way then if you are ok-ish and since it is only about opacity :yum: